steam_desktop_updater.py

```diff
@@ -178,15 +174,28 @@ def extract(self, datadir: Path):
             subimg.save(self.get_dest(h, datadir), format='PNG')
 
 
+def _get_folder_paths(steam_root: Path, library_folders: dict):
+    paths = [steam_root]
+    for k, v in library_folders.items():
+        if not k.isdigit():
+            continue
+        if isinstance(v, dict) and "path" in v:
+            paths.append(Path(v["path"]))
+        else:
+            paths.append(v)
```

Comment:
`paths.append(Path(v))` probably? `v` is a string here.

Reply:
Yes it should, fixed